src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1472,6 +1472,9 @@ public class ClusterVNode<TStreamId> :
 				_mainBus.Subscribe<ReplicationMessage.AckLogPosition>(replicaService);
 				_mainBus.Subscribe<ClientMessage.TcpForwardMessage>(replicaService);
 			}
+			else {
+				monitoringInnerBus.Subscribe<ReplicationMessage.GetReplicationStats>(leaderReplicationService);
```

Comment:
It might be worth adding a comment here about the LeaderReplicationService only running on a single node to provide stats, which is why it's not subscribed to the other message types (e.g. SystemStart and StateChangeMessage)

Reply:
Done.